### PR TITLE
ceph-disk: create deprecation warnings

### PR DESCRIPTION
**Pull request by alfredodeza**
Adds a banner-like deprecation warning for ceph-disk, and a doc section to help ceph-disk users what the correct way of migrating or consuming ceph-volume would be, with the noted exception of encryption

### CODE REVIEW

File: doc/ceph-volume/index.rst [hunk 1]
```diff
@@ -17,6 +17,33 @@ that may have been deployed with ``ceph-disk``.
 * :ref:`ceph-volume-lvm`
 * :ref:`ceph-volume-simple`
 
+
+migrating
```
Comment (liewegas) [inline doc/ceph-volume/index.rst]: Migrating

File: src/ceph-disk/ceph_disk/main.py [hunk 2]
```diff
@@ -5658,10 +5671,20 @@ def main(argv):
     CEPH_PREF_GROUP = args.setgroup
 
     if args.verbose:
-        args.func(args)
+        try:
+            args.func(args)
+        except Exception:
+            # warn on any exception when not running with verbosity
```
Comment (liewegas) [inline src/ceph-disk/ceph_disk/main.py]: is this backwards?  this looks like the verbose path

Comment (alfredodeza) [reply src/ceph-disk/ceph_disk/main.py]: Yes, my wording is backwards, hah!

**Comment by smithfarm** [comment]
@liewegas Should we backport this to luminous?

**Comment by andrewschoen** [comment]
@smithfarm it was already backported here https://github.com/ceph/ceph/pull/18989

**Comment by smithfarm** [comment]
@andrewschoen OK. Would it make sense to add something to `PendingReleaseNotes`?

**Comment by andrewschoen** [comment]
@smithfarm sure, that sounds reasonable.

**Pull request merged**